buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -1259,6 +1259,16 @@ def set_build_extra_options(build_specs=null) {
         if (!EXTRA_MAKE_OPTIONS) {
             EXTRA_MAKE_OPTIONS = buildspec.getVectorField("extra_make_options", SDK_VERSION).join(" ")
         }
+        def cmake_opts = []
+        // Wrap each option in quotes
+        buildspec.getVectorField("extra_cmake_options", SDK_VERSION).each { opt ->
+            cmake_opts << "\"${opt}\""
+        }
+        if(cmake_opts){
+            cmake_opts = cmake_opts.join(" ")
+            // Put the cmake options at the beginning, so that user can override them with the EXTRA_MAKE_OPTIONS if needed
+            EXTRA_MAKE_OPTIONS = "'EXTRA_CMAKE_OPTIONS=${cmake_opts}' " + EXTRA_MAKE_OPTIONS
```

Comment:
The single quotes here are new.
See [new build param](https://ci.eclipse.org/openj9/job/Build_JDK11_x86-64_linux_cm_Personal/91/parameters/) vs [old build param](https://ci.eclipse.org/openj9/job/Build_JDK11_x86-64_linux_cm_Personal/90/parameters/)
`'EXTRA_CMAKE_OPTIONS="-DOMR_WARNINGS_AS_ERRORS=FALSE"' `
`EXTRA_CMAKE_ARGS="-DOMR_WARNINGS_AS_ERRORS=FALSE"`
I assume that is intended but just want to confirm. This also causes the double quotes to live longer.
Old make command
`make EXTRA_CMAKE_ARGS=-DOMR_WARNINGS_AS_ERRORS=FALSE all`
New make command
`make 'EXTRA_CMAKE_OPTIONS="-DOMR_WARNINGS_AS_ERRORS=FALSE"' 'EXTRA_CMAKE_OPTIONS="-DOMR_WARNINGS_AS_ERRORS=FALSE"' all`
As you can see the option also gets doubled up. This is partially because of this PR and partially with how the options are parsed. The Pipeline level [calls](https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/common/variables-functions.groovy#L973) `set_build_extra_options` which sets the option and then the build uses the param and also [calls](https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/common/variables-functions.groovy#L963) `set_build_variables` which [calls](https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/common/variables-functions.groovy#L583) `set_build_extra_options` and repeats the option. I'm not sure why we call it twice or whether it is intended or not.